### PR TITLE
Fix a few notes

### PR DESCRIPTION
**Pull request by torarnv**

### CODE REVIEW

File: src/docs/product/cli/dif.mdx [hunk 1]
```diff
@@ -9,14 +9,12 @@ information about crash reports for most compiled platforms. `sentry-cli` can be
 used to validate and upload debug information files. For more general
 information, refer to [_Debug Information Files_](/workflow/debug-files/).
 
-<Note>
-
-Source maps, while also being debug information files, are handled differently
 
+<Note>
+Source maps, while also being debug information files, are handled
+differently in Sentry. For more information see [Source Maps in sentry-cli](/product/cli/releases/#sentry-cli-sourcemaps).
```
Comment (torarnv) [inline src/docs/product/cli/dif.mdx:15]: Hmm, the <Note> does not seem to enjoy having links inside it.

https://sentry-docs-f3z0ufimj.sentry.dev/product/cli/dif/

Comment (torarnv) [reply src/docs/product/cli/dif.mdx:15]: Fixed by adding spaces before and after note tag

**Comment by torarnv** [comment]
Ready to be reviewed, CC @bruno-garcia

**Review by bruno-garcia — APPROVED** [review]
Thanks @torarnv !

**Pull request merged**